added docs restructuring (#81)
dappforce/subsocial-docs: docs/Bounties.md

---
id: bounties
title: Developer Bounties
displayed_sidebar: developSidebar
---

### Hey Devs 👋🏼

Planning to build something on Subsocial? 

> We have a set of bounties we would love you to build. 

Reviewer: ```suggestion
We have a set of bounties we would love for you to build. 
```

:::info NOTE

> Before starting to build, join our [Telegram](https://t.me/+ZzvLu0ZfkQwxNGQy) & [Discord](https://discord.gg/AnfSdfjy) channels to discuss with us. 

Reviewer: ```suggestion
Before starting to build, join our developer channels on [Telegram](https://t.me/+ZzvLu0ZfkQwxNGQy) & [Discord](https://discord.gg/MSW4xuvTE2) to discuss with us. 
```
Previous discord link had expired, this one will last forever

:::

## Bounties 💰
### Browser Extension to Resolve Subsocial Domains
- Subsocial allows people to register domains at https://polkaverse.com/dd

> - Domains have several values attached to them, like spaces on the network, URL of projects, etc. 

Reviewer: ```suggestion
- Domains have several values attached to them, like spaces on the network, URLs of projects, etc. 
```

> - We want you to create a browser extension that will resolve these domains from the browser’s search bar and navigate user directly to the linked website. 

Reviewer: ```suggestion
- We want you to create a browser extension that will resolve these domains from the browser’s search bar and navigate users directly to the linked website. 
```

> - The extension should show the list of domains owned by an address. Ability to save your address to see and explore the domains you own.

Reviewer: ```suggestion
- The extension should show the list of domains owned by an address, and let you save your address in order to see and explore the domains you own.
```

> - It needs to mark whether a domain is parked free or owned by a particular address. If available, then redirect to domain buy page.

Reviewer: ```suggestion
- It needs to mark whether a domain is available, or owned by a particular address. If available, then redirect to the domain buy page.
```

> - You can use Subsocial JS SDK and our docs to do it.

Reviewer: ```suggestion
- You can use the Subsocial JS SDK and our docs to do it.
```

## Hackathons 🧑🏻‍💻

### IBC Media Hack - 2

> [IBC Media](https://ibc.media/hackathon-2) is a open to all hackathon organized in Hybrid Mode in India and Remotely. 

Reviewer: ```suggestion
[IBC Media](https://ibc.media/hackathon-2) is a hybrid open-to-all hackathon that takes place in India and online. 
```

> We are sponsoring for the following bounties:

Reviewer: ```suggestion
We are sponsoring the following bounties:
```

> #### Browser Extension to Resolve Subsocial Domains

Reviewer: copy the above section after making the changes
